numpy/core/tests/test_multiarray.py

```diff
@@ -4672,6 +4672,35 @@ def test_load_object_array_fromfile(self):
         assert_raises_regex(ValueError, "Cannot read into object array",
                             np.fromfile, self.filename, dtype=object)
 
+    def test_fromfile_offset(self):
+        with open(self.filename, 'wb') as f:
+            self.x.tofile(f)
+
+        with open(self.filename, 'rb') as f:
+            y = np.fromfile(f, dtype=self.dtype, offset=0)
+            assert_array_equal(y, self.x.flat)
+
+        with open(self.filename, 'rb') as f:
+            offset_items = len(self.x.flat) // 4
+            offset_bytes = self.dtype.itemsize * offset_items
```

Comment:
nit: I think it would be clearer to distinguish `offset_items` and `count_item` here, rather than using the same variable for both.